lablup/backend.ai-webui
# update: pnpm-lock file to apply  properly

This PR resolves [#2959](https://github.com/lablup/backend.ai-webui/issues/2959).


### Test
> Prerequisites: you need to install pnpm in target machine. (Install guide: https://pnpm.io/installation)

1. Execute command `pnpm i` 
2. Check whether installation is successfully done or not.

**Checklist:** (if applicable)
